packages/plugin-svgo/src/config.js

```diff
@@ -1,3 +1,4 @@
+import { extendDefaultPlugins } from 'svgo';
```

Comment:
The latest svgo version provides preset-default plugin. extendDefaultPlugins is deprecated and will trigger warning.